__tests__/index.test.js

```diff
@@ -2,39 +2,37 @@ const nock = require('nock');
 const cli = require('../src');
 const { version } = require('../package.json');
 const conf = require('../src/lib/configstore');
-const swaggerCmd = require('../src/cmds/swagger');
 
 describe('cli', () => {
-  it('command not found', () => {
-    expect.assertions(1);
```

Comment:
Changed these to use `.rejects` and `.resolves` because calling `expect.assertions()` or `expect.hasAssertions()` has always been a bit gnar.